2019/12/27/from-dev-to-pm.html.md

```diff
@@ -0,0 +1,233 @@
+family, their business, their religion .. politics... ok enough already. But believe it or not,
+all of the aforementioned points will come into play with a long term client. But we want to talk
+about the basics here. We will assume we have a first time client, with a first time (to us)
+project.
+
+Find out about the clients' business. How do they make money with this business? While you are
+talking to the client about this, ask questions that get the client to explain and promote
+their business to you. For most, this will be easy... sometimes to the point of having to learn
+how to politely stop them!  But you will learn far more about their business, and their personal
+approach to that business by *listening*.. than you will by talking. Just get them started, and let
+them roll. In the early meetings, ask questions about why and how the client feels their
+product / service is superior to others.
+Find out who in the company was responsible for various aspects of the product / service. Learn who
+the stakeholders are. Try not to ask too many technical questions until the project outline has
+been defined, and you personally have a good feel for what it is comprised of.
+
+**Clients are not free**
+
+Clients are a long term (and ongoing) proposition. Take care of them and they will return
+again and again.  Ignore them and they will respond in kind, and usually very quickly. The amount
+of time you spend with client relationships is normally non-billable. So keep that in mind the
+next time a client is hard to deal with and you think "We don't need clients like this, lets
+just find a new one". You will invest a lot of time and energy with a client.
+Try not to waste it.
+</pre>
+### 2. Learn your $project
+<pre>
+If you have a good understanding of the client and their business, you next want them to explain
+to you, what they are attempting to accomplish with this project. Let me clarify here, you do NOT
+want them to explain how they want to do a project, you want to first learn **what the client
+wants to accomplish with the project**.
+
+This is important, and should be <u>understood completely</u> ... because if you blow this phase
+you can easily miss one of the major points of e-commerce project management. I have found that
+about 7 of 10 clients have major misconceptions about how things actually work in our business.
+They usually have a very good idea of **what** they want to have happen in the end, but not so
+much **how** to make that happen.
+
+When I say <u>understand completely</u>, I am talking about the 3 of 10 that DO know what they are
+talking about and **need** to be taken literally.
+
+It will be up to you to determine what type of client you are dealing with. Use your past
+experiences with people to figure this out. It can be hard. You have to make sure you don't insult
+someone's intelligence, and at the same time you need to know if they have it!  I have found it's
```

Comment:
Since "insult intelligence" is broad, and we're talking about something specific here: `s/if they have it/if they have this kind of experience specifically/`

Reply:
Reworded